encoders/nlp/TFIDFTextEncoder/__init__.py

```diff
@@ -0,0 +1,34 @@
+__copyright__ = "Copyright (c) 2021 Jina AI Limited. All rights reserved."
+__license__ = "Apache-2.0"
+
+import numpy as np
+import torch
+
+
+class TFIDFTextEncoder(BaseEncoder):
+    """Encode data from a `np.ndarray` (of strings) of length `BatchSize` into
+    a `csr_matrix` of shape `Batchsize x EmbeddingDimension`. 
+
+    :param path_vectorizer: path containing the fitted tfidf encoder object
+    :param args: not used
+    :param kwargs: not used
+    """
+
+    def __init__(self,
+                 path_vectorizer= "./pods/tfidf_vectorizer.pickle",
+                 *args,
+                 **kwargs):
+        super().__init__(*args, **kwargs)
+        self.path_vectorizer = path_vectorizer
+
+    def post_init(self):
```

Comment:
call super `post_init`

Reply:
Don't know what you mean...

Reply:
u need to call the post init of the super class.

Reply:
```suggestion
    def post_init(self):
         super().post_init()
```